lib/features/local_live_log.js

```diff
@@ -164,6 +164,14 @@ export default class TaskclusterLogs {
 
   async killed(task) {
     debug('switching live log redirect to backing log...')
+    try {
+      await this.container.stop();
```

Comment:
Before the container would just keep running until it was explicitly marked to be removed.